src/Lint.jl

```diff
@@ -345,31 +345,36 @@ function lintdir{T<:AbstractString}(dir::T, ctx::LintContext=LintContext())
     ctx.messages
 end
 
+function readandwritethestream(conn)
+    println("Connection accepted")
+    # Get file, code length and code
+    file = strip(readline(conn))
+    println("file: ", file)
+    code_len = parse(Int, strip(readline(conn)))
+    println("Code bytes: ", code_len)
+    code = Compat.UTF8String(read(conn, code_len))
+    println("Code received")
+    # Do the linting
+    msgs = lintfile(file, code)
+    # Write response to socket
+    for i in msgs
+        write(conn, string(i))
+        write(conn, "\n")
+    end
+    # Blank line to indicate end of messages
+    write(conn, "\n")
+    println("Connection closed")
+    close(conn)
```

Comment:
Sorry, I didn't notice your comments earlier. This is the whole point of this pull request. Unless this connection is properly closed the coffeescript code keeps hanging. I don't have the expertise on server client coding, but this is based on observing what is happening in practice. The same happens with tests/server.jl if you comment out the close(conn) here, which gives you possibility to test it yourself easily.